platform.go

```diff
@@ -92,18 +95,40 @@ var (
 // SupportedPlatforms returns the full list of supported platforms for
 // the version of Go that is
 func SupportedPlatforms(v string) []Platform {
-	if strings.HasPrefix(v, "go1.0") {
-		return Platforms_1_0
-	} else if strings.HasPrefix(v, "go1.1") {
-		return Platforms_1_1
-	} else if strings.HasPrefix(v, "go1.3") {
-		return Platforms_1_3
-	} else if strings.HasPrefix(v, "go1.4") {
-		return Platforms_1_4
-	} else if strings.HasPrefix(v, "go1.5") {
+	// go-version only cares about version numbers
+	if strings.HasPrefix(v, "go") {
+		v = v[2:]
+	}
```

Comment:
Actually I was thinking if `!HasPrefix` we bail with the default here.

Reply:
good call